### PR TITLE
Refactored test assertions that have suboptimal tests with numbers

### PR DESCRIPTION
**Pull request by freddiewanah**
### Description

As discussed in PR #7609, I tried to break the suboptimal test refactors to smaller pieces.

Suboptimal Assert: Instead of using statements such as assertIsNone, assertIsInstance, always simply use assertTrue or assertFalse. This will decrease the code overall readability and increase the execution time as extra logic needed.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: tests/test_bundle_get_data.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def test_get_all_bundles_list(self, params):
             output = get_all_bundles_list(**params)
             self.assertTrue(isinstance(output, list))
             self.assertTrue(isinstance(output[0], tuple))
-            self.assertTrue(len(output[0]) == 2)
+            self.assertEqual(output[0], 2)
```
Comment (KumoLiu) [inline tests/test_bundle_get_data.py:56]:
```suggestion
            self.assertEqual(len(output[0]), 2)
```

Looks like len should be added here?

**Review by KumoLiu — APPROVED** [review]
Thanks for the PR , overall looks good to me.
Just a minor comment inline.

File: tests/test_bundle_get_data.py (hunk 1 above)
Comment (freddiewanah) [reply tests/test_bundle_get_data.py]: You're absolutely right, and sorry for making this mistake.

**Pull request merged**